src/test/lib/storage/buffer/page_id_test.cpp

```diff
@@ -0,0 +1,34 @@
+#include "base_test.hpp"
+#include "storage/buffer/page_id.hpp"
+
+namespace hyrise {
+
+class PageIDTest : public BaseTest {};
+
+TEST_F(PageIDTest, TestPageIDForDifferentSizeTypes) {
+  const auto small_page = PageID{PageSizeType::KiB16, 29, true};
+  EXPECT_EQ(small_page.size_type(), PageSizeType::KiB16);
+  EXPECT_EQ(small_page.byte_count(), 16384);
+  EXPECT_TRUE(small_page.valid());
+  EXPECT_EQ(small_page.index(), 29);
+
+  const auto larger_page = PageID{PageSizeType::KiB256, 59, true};
+  EXPECT_EQ(larger_page.size_type(), PageSizeType::KiB256);
+  EXPECT_EQ(larger_page.byte_count(), 262144);
+  EXPECT_TRUE(larger_page.valid());
+  EXPECT_EQ(larger_page.index(), 59);
+}
+
+TEST_F(PageIDTest, TestInvalidPageID) {
+  EXPECT_FALSE(PageID(PageSizeType::KiB16, 29, false).valid());
+  EXPECT_FALSE(INVALID_PAGE_ID.valid());
+}
```

Comment:
Could you please add some test cases testing the PageID's equality operator?
This might include (i) a valid page id, (ii) the comparison between an invalid page ID with a page size type > MIN_PAGE_SIZE and `INVALID_PAGE_ID`, and (ii) the comparison between an invalid page ID with an index > 0 and `INVALID_PAGE_ID`.